docs/API/InstrumentFormat.md

```diff
@@ -30,7 +30,11 @@ Instrument format:
         "InstrumentVersion": string,
         "InstrumentFormatVersion" : "v0.0.2-dev",
         "ShortName" : "InstrumentName", /* Required */
-        "LongName"  : "The Human Readable Instrument Name", /* Required */
+        "LongName": {
+          "en-ca": "The Human Readable Instrument Name",
+          "fr-ca": "Bonjour"
+        }, /* An object keyed by language - Required */
+        "SupportedLanguages": array(string),
```

Comment:
`array()` is PHP, not JSON. It should be `[]`. The values for "string" should also be defined.

(Is this even required? Can't it be deduced from the options in longname?)